pennylane/workflow/qnode.py

```diff
@@ -111,6 +111,36 @@ def _to_qfunc_output_type(
     return type(qfunc_output)(results)
 
 
+def _validate_gradient_kwargs(gradient_kwargs: dict) -> None:
+    for kwarg in gradient_kwargs:
+        if kwarg == "expansion_strategy":
+            raise ValueError(
+                "expansion_strategy is no longer a valid keyword argument to QNode."

```

Comment:
```suggestion
                "'expansion_strategy' is no longer a valid keyword argument to QNode."

```
Just for symmetry with the 'shots' message below

Reply:
adding locally.